ports/mimxrt10xx/apps/esp32programmer/main.c

```diff
@@ -38,95 +38,151 @@
  * used to program ESP32 Co-Processors
  */
 
+// Enable this for more reliable connection require esptool.py default reset option "--before default_reset"
+// i.e "--before no_reset" should not be include in the esptool.py command
+#define ESP32_DTR_RTS_BOOT_RESET_SUPPORT    0
```

Comment:
DTR/RTR signal manipulation with esptool.py is supported but not enabled to be compatible with `--before no_reset` option used in adafruit learn guide/tutorial. Can be enabled if needed.